src/types.h

```diff
@@ -38,6 +38,7 @@ enum class Type
   inet,
   stack_mode,
   array,
+  ctx,
```

Comment:
I think a comment above each type explaining what they're for would be useful going forward. No need to do the others but could you add one for "ctx" please?